layers/+lang/javascript/packages.el

```diff
@@ -260,6 +261,38 @@
           "sR" 'spacemacs/skewer-eval-region-and-focus
           "ss" 'skewer-repl)))))
 
+(defun javascript/init-indium ()
+  (when (eq javascript-repl 'indium)
+    (use-package indium
+      :defer t
+      :init
+      (progn
+        (require 'indium)
+        (add-hook 'js2-mode-hook 'indium-interaction-mode)
+        ;; prefixes
+        (spacemacs/declare-prefix-for-mode 'indium-interaction-mode "mdi" "debugger")
```

Comment:
Do you really want to set a prefix for `mdi` I would have expected to have a prefix for `md` instead for debugging.
Also you need to make sure not to bind this when `dap` is activated i.e.
```elisp
(pcase (spacemacs//javascript-backend)
    (`lsp (yourBindings)))
```